lib/ABI/FunctionType/Layout.cpp

```diff
@@ -342,6 +346,19 @@ ToRawConverter::finalStackOffset(uint64_t SizeOfArgumentsOnStack) const {
   return Result;
 }
 
+uint64_t ToRawConverter::combinedStackArgumentSize(CFT Function) const {
```

Comment:
I'd prefer to have `using TRC = ToRawConverter` rather than `using CFT =`.

Reply:
Done.